### ISSUE TITLE
V6 RC2 - change prop is not bound to dispatch

### ISSUE THREAD

**Issue by epeli** [erikras/redux-form#1247]
The [documentation](http://redux-form.com/6.0.0-alpha.15/docs/api/ActionCreators.md/) states that the action creators are bound to the Redux `dispatch` but it doesn't seem to be the case with `change` function.

This does not work

``` js
this.props.change("fieldName", "new value");
```

But this works

``` js
this.props.dispatch(this.props.change("fieldName", "new value"));
```

**Comment by dmeenhuis** [comment]
I've updated to RC2 this morning and it seems this problem still exists.

**Comment by wesleycyu** [comment]
In `6.0.0-alpha.6` even `this.props.dispatch(this.props.change("fieldName", "new value"));` doesn't work.

This is because `change` is incorrectly returning
```
{
   type: "redux-form/CHANGE",
   value: "fieldName",
   touch: false,
   form: "formName"
}
```

Instead of:

```
{
   type: "redux-form/CHANGE",
   field: "fieldName",
   value: "new_value",
   touch: false,
   form: "formName"
}
```

It's missing the `field` attribute and mis-assigning the `value` attribute.

**Comment by yormi** [comment]
Ok guys ! After spending to much time on my problem, I finally end-up with the same conclusion with `v6.0.1`:
* The change is not bound to dispatch as written in the doc but instead returns the action.
* However, the action has been fixed regarding the mis-assigning mentionned by @wesleycyu !

**Comment by Philipp91** [comment]
Would be nice if "change" was indeed a bound action creator as the documentation says.

**Comment by dtipson** [comment]
Yep, focus appears to be missing in the actionsCreators passed to the form props.

**Comment by gustavohenke** [comment]
It seems like this was fixed around v6.0.4.
Tested in the examples and it worked.
